Makefile

```diff
@@ -16,10 +16,31 @@
 # Author: Shawn Morel (shawnmorel@gmail.com)
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 
-# Cockroach build rules.
-GO ?= go
-# Allow setting of go build flags from the command line.
-GOFLAGS :=
+# Variables to be overridden in the environment or on the command line, e.g.
+#
+#   GOFLAGS=-msa make build
+GO      ?= go
+GOFLAGS ?=
+
+# Variables to be overridden on the command line only, e.g.
+#
+#   make test PKG=./pkg/storage TESTFLAGS=--vmodule=raft=1
+BENCHES      := -
```

Comment:
I'm not a fan of alphabetizing these (for example, it was good to have the three timeouts together, or `TESTS` and `BENCHES`), but whatever.